tests/test_dimension_associations.py

```diff
@@ -0,0 +1,58 @@
+import shutil
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from dsgrid.config.dimension_associations import DimensionAssociations
+from dsgrid.dimension.base_models import DimensionType
+
+
+DATA_DIR = Path("tests") / "data" / "dimension_associations"
+
+
+def test_dimension_associations(spark_session):
+    da = DimensionAssociations.load(Path("."), DATA_DIR.iterdir())
+    assert da.has_associations(DimensionType.SECTOR, DimensionType.SUBSECTOR)
+    assert not da.has_associations(DimensionType.SECTOR, DimensionType.TIME)
```

Comment:
Even though we don't provide an association table for this, I still want to be able to see which Sectors are in each Geography (or Time). Remember that if it isn't provided, then we assume a full join between the combos of dimensions.